app/services/reporting/account_reuse_report.rb

```diff
@@ -133,8 +335,12 @@ def num_active_profiles
 
     def params
       {
-        query_date: report_date.end_of_day,
+        query_date: first_day_of_report_month,
       }.transform_values { |v| ActiveRecord::Base.connection.quote(v) }
     end
+
+    def first_day_of_report_month
+      report_date.beginning_of_month.strftime('%Y-%m-%d')
+    end
```

Comment:
we shouldn't need to `strftime` this if we're passing it to `quote()` for the DB